parse-nessus-xml.py

```diff
@@ -213,14 +213,21 @@ def remediation_plan(vuln):
                 print('\t{}'.format(site))
 
 print("\n-------  CSV  ------\n")
+
+weakness_desc = sched_completion_date = milestone = \
+    deviation_rationale = supporting_docs = comments = auto_approve = ""
+KEV = "No"
+
 for vuln in sorted(vuln_report):
     if vuln_report[vuln]["risk_factor"] != "None":
         number_of_affected_hosts = len(vuln_report[vuln]["hosts"])
         risk_factor = vuln_report[vuln]["risk_factor"] 
         if risk_factor == "Medium" :
             risk_factor = "Moderate"
         weakness_name=vuln_report[vuln]["plugin_name"]
-        csvwriter.writerow(["CGXX","RA-5", weakness_name, "", "Nessus Scan Report", 
+        csvwriter.writerow(["CGXX","RA-5", weakness_name, weakness_desc, "Nessus Scan Report", 
             vuln_report[vuln]["id"], str(number_of_affected_hosts) + " production hosts", 
-            owner, "None", remediation_plan(weakness_name), start_date.date(), "", "Resolve", "", mmddYY, "Yes", mmddYY,
-            "CloudFoundry stemcell", risk_factor, risk_factor, "No", "No", "No" ])
+            owner, "None", remediation_plan(weakness_name), start_date.date(), sched_completion_date, 
+            "Resolve", milestone, mmddYY, "Yes", mmddYY,
+            "CloudFoundry stemcell", risk_factor, risk_factor, "No", "No", "No",
+            deviation_rationale, supporting_docs, comments, auto_approve, KEV])
```

Comment:
this is getting hard to parse in my brain.
These names feel like a step in the right direction, but I'm still having a hard time holding enough of this list in context long enough to compare the diff

Reply:
Would just splitting it to multiple lines (one entry per line) help?

Reply:
Next iteration (if any) I'll make it one entry per line.